nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/java/org/apache/nifi/lookup/RestLookupService.java

```diff
@@ -172,6 +173,20 @@ public class RestLookupService extends AbstractControllerService implements Reco
         .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
         .build();
 
+    static final AllowableValue PASS_THROUGH = new AllowableValue("Pass-through",
+            "Pass-through","sends successful and unsuccessful HTTP responses to the caller");
+    static final AllowableValue HANDLE = new AllowableValue("Handle Service Errors",
+            "Handle Service Errors", "generates an exception when an unsuccessful HTTP response code is received");
```

Comment:
Declaring these options as an `enum` that implements `DescribedValue` makes it easier to refer to the parsing value of the property. Recommend naming the enum `ResponseHandlingStrategy`.

Reply:
Sounds good.  This change has been incorporated in the latest commit.